sums.rb

```diff
@@ -0,0 +1,22 @@
+class Sum1
+    attr_accessor:total
+    def initialize(a,b)
+        @total=a+b
+    end
+end
+
+class Sum2
+    def initialize(p1,p2)
+        @a=p1
+        @b=p2
+    end
+    def new_total
+        return @a+@b
+    end
+end
+
+sum1=Sum1.new(5,6)
+sum2=Sum2.new(5,6)
+
+print sum1.total
+print sum2.new_total
```

Comment:
This was just printing `1111`, so again it's difficult for the user to distinguish the two results. Going forward, please verify that the output is easy to read, or I may ask you to fix it and re-submit.

```suggestion
puts sum1.total
puts sum2.new_total
```